python-test/features/steps/control_plane_policies.py

```diff
@@ -137,3 +191,28 @@ def delete_policy(token, policy_id):
     assert_that(response.status_code, equal_to(204), 'Request to delete policy id='
                 + policy_id + ' failed with status=' + str(response.status_code))
 
+
+def check_logs_contain_message_for_policies(logs, expected_message, policies_list,
+                                          timestamp_considered=datetime.now().timestamp()):
+    """
+    Checks agent container logs for expected message for all applied policies
+
+    :param (list) logs: list of log lines
+    :param (str) expected_message: message that we expect to find in the logs
+    :param (str) policies_list: list with all policy id applied to the agent
+    :param timestamp_considered: timestamp from which the log will be considered.
+                                                                Default: timestamp at which behave execution is started
+    :returns: (bool) whether expected message was found in the logs for expected policy
+
+
+    """
+    policies_with_message = set()
+    for log_line in logs:
+        log_line = safe_load_json(log_line)
+        if log_line is not None and log_line['msg'] == expected_message:
+            for policy_id in policies_list:
+                if 'policy_id' in log_line.keys() and log_line['policy_id'] == policy_id and log_line['ts'] > timestamp_considered:
```

Comment:
These `if` statements are fuzzy and made the code confusing.
Let's create a helper function `isExpectedLogLine()` in this module that tests log lines.
Log lines should:
- not be None
- have a 'msg' property that matches the `expected_message` string.
- Have a 'ts' property whose value is greater than `timestamp_considered`
- have a property 'policy_id' that is also contained in the `policies_list` list.